libs/template/presets/storefront/experience/pages/my-account/carts-page.ts

```diff
@@ -0,0 +1,68 @@
+import { ExperienceComponent } from '@spryker-oryx/experience';
+import { RouteType } from '@spryker-oryx/router';
+import { Size } from '@spryker-oryx/utilities';
+
+export const cartsPage: ExperienceComponent = {
```

Comment:
We're on are way to move experience data as much as possible to the domains, and only let the presets to add the actual page. Can you follow the example of merchant here?